scene/2d/tile_map.h

```diff
@@ -178,7 +178,7 @@ class TileMap : public Node2D {
 	float bounce;
 	uint32_t collision_layer;
 	uint32_t collision_mask;
-	DataFormat format;
+	mutable DataFormat format;
```

Comment:
What does this do? Aren't all properties mutable unless you make them `const` in C++?

Reply:
The get_tile_data() function is `const`, meaning it can't change member variables of the class (https://stackoverflow.com/questions/3141087/what-is-meant-with-const-at-end-of-function-declaration). The `mutable` keyword on a variable in the class overrides that for that variable, so now `format` can be modified from within `const` functions

Reply:
oh, now i remember why i didn't write that line in the get_tile_data function, didn't know about the mutable keyword usage :smile:

Reply:
Haha! Well im glad I could teach y'all something. 😂